src/components/transactions/TxDetails/TxData/DecodedData/MethodCall.tsx

```diff
@@ -0,0 +1,55 @@
+import NamedAddressInfo from '@/components/common/NamedAddressInfo'
+import { Box, Typography } from '@mui/material'
+
+const MethodCall = ({
+  method,
+  contractAddress,
+  contractName,
+  contractLogo,
+}: {
+  method: string
+  contractAddress: string
+  contractName?: string
+  contractLogo?: string
+}) => {
+  return (
+    <>
+      <Typography fontWeight="bold" display="flex" alignItems="center" gap=".5em" component="div">
+        Call
+        <Typography
+          component="code"
+          variant="body2"
+          sx={{
+            backgroundColor: 'background.main',
+            px: 1,
+            py: 0.5,
+            borderRadius: 0.5,
+            fontFamily: 'monospace',
+            whiteSpace: 'nowrap',
+          }}
+        >
+          {method}
+        </Typography>{' '}
+        on
+        <NamedAddressInfo
+          address={contractAddress}
+          name={contractName}
+          customAvatar={contractLogo}
+          onlyName
+          hasExplorer
+          showCopyButton
+          avatarSize={24}
+        />
+      </Typography>
+
+      {/* Divider */}
+      <Box
+        borderBottom="1px solid var(--color-border-light)"
+        width="calc(100% + 32px)"
+        sx={{ ml: '-16px !important' }}
+      />
```

Comment:
The way we divide content is a bit misleading sometimes. For instance in this transaction:
![Screenshot 2024-07-25 at 11 50 03](https://github.com/user-attachments/assets/c0c63a88-745c-4adc-bc0a-1f2a79d302d3)

This divider separates the Method Call from the Parameters (No Parameters). But there is no Divider splitting the parameters from the Advanced Details and no Divider splitting the Info text (Sign or immediately execute this transaction) from the Method Call.
This looks like the Parameters are part of the Details and Balance Change block - even though they are in fact a part of the Method Call.

Reply:
I've put it in a rounded border to differentiate from the other blocks.

<img width="708" alt="Screenshot 2024-07-25 at 12 59 42" src="https://github.com/user-attachments/assets/18691226-078b-4022-a8a0-ecf17b050738">